package.json

```diff
@@ -211,7 +211,7 @@
     "cypress": "^10.9.0",
     "cypress-grep": "^3.0.3",
     "cypress-real-events": "^1.7.1",
-    "documentation": "^13.2.5",
+    "documentation": "^14.0.0",
```

Comment:
Maybe you should add a comment something like this
```js
// @ts-expect-error There is a type mismatch after upgrading `documentation` package.
```

above

https://github.com/metabase/metabase/blob/e585efaa8cd7e931678f5bbec8981801bcfe21f7/frontend/src/metabase/core/components/Markdown/Markdown.tsx#L14

since the dependencies upgrade cause the type error.

And I briefly test our Markdown implementation and it still looks the same as before so this PR should be good to go.

#### After
![Screenshot 2022-10-26 at 12 05 58 PM](https://user-images.githubusercontent.com/1937582/197939926-54ae1d7e-9511-4fcd-8fa2-4066966d51b1.png)

#### Before
![Screenshot 2022-10-26 at 12 04 27 PM](https://user-images.githubusercontent.com/1937582/197939944-b27e8060-37e0-4c92-ad6b-4261999f21f2.png)

Reply:
thanks for the review! where did you see the type mismatch error?

Reply:
@paoliniluis If you click on the **Files changed** tab or visit this link https://github.com/metabase/metabase/pull/26077/files. Then scroll all the way to the bottom you'll see this.
![image](https://user-images.githubusercontent.com/1937582/198172468-4621d227-8b86-4272-b718-47893f1b7a66.png)

I've verified that `master` doesn't have this problem.

Reply:
Thanks for the explanation. This is extremely odd, documentation is a devdependency while our Markdown implementation should be completely out of that. Maybe it's a false positive?

Reply:
With `documentation` upgrade, if I remember correctly, it requires the new version of `remark-gfm`, it was `3.0.1` something I don't exactly remember. But we have version `1.0.0` pinned in `package.json`. So, I think the type is now conflicting because of that.
https://github.com/metabase/metabase/blob/8a339362176f5351d972c5372dd722eb6139f678/package.json#L96

<details>
  <summary>Output from `yarn why remark-gfm`</summary>

  ```
  yarn why remark-gfm
  yarn why v1.22.17
  [1/4] 🤔  Why do we have the module "remark-gfm"...?
  [2/4] 🚚  Initialising dependency graph...
  warning Resolution field "ansi-regex@5.0.1" is incompatible with requested version "ansi-regex@^2.0.0"
  warning Resolution field "set-value@4.0.1" is incompatible with requested version "set-value@^2.0.0"
  warning Resolution field "set-value@4.0.1" is incompatible with requested version "set-value@^2.0.1"
  warning Resolution field "prismjs@1.27.0" is incompatible with requested version "prismjs@~1.25.0"
  warning Resolution field "nth-check@2.0.1" is incompatible with requested version "nth-check@^1.0.2"
  warning Resolution field "ansi-regex@5.0.1" is incompatible with requested version "ansi-regex@^2.0.0"
  warning Resolution field "ansi-regex@5.0.1" is incompatible with requested version "ansi-regex@^4.0.0"
  warning Resolution field "ansi-regex@5.0.1" is incompatible with requested version "ansi-regex@^6.0.1"
  warning Resolution field "ansi-regex@5.0.1" is incompatible with requested version "ansi-regex@^2.0.0"
  warning Resolution field "nanoid@3.1.31" is incompatible with requested version "nanoid@^3.3.3"
  [3/4] 🔍  Finding dependency...
  [4/4] 🚡  Calculating file sizes...
  => Found "remark-gfm@1.0.0"
  info Has been hoisted to "remark-gfm"
  info This module exists because it's specified in "dependencies".
  info Disk size without dependencies: "28KB"
  info Disk size with unique dependencies: "372KB"
  info Disk size with transitive dependencies: "4.56MB"
  info Number of shared dependencies: 38
  => Found "documentation#remark-gfm@3.0.1"
  info This module exists because "documentation" depends on it.
  info Disk size without dependencies: "28KB"
  info Disk size with unique dependencies: "712KB"
  info Disk size with transitive dependencies: "5.19MB"
  info Number of shared dependencies: 47
  ```
</details>

You'd see from the output that there's this line `Found "documentation#remark-gfm@3.0.1"`

A quick Google search returns some seemingly related threads
- https://github.com/remarkjs/react-markdown/issues/627
- https://github.com/orgs/rehypejs/discussions/63
Which tell you to just ignore the error.

Reply:
thanks, I just tested and it works without issues :)